project-clone/rhel.Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build the manager binary
+FROM registry.redhat.io/rhel8/go-toolset:1.13.4-27 as builder
```

Comment:
can't we avoid having two dockerfiles?

Reply:
Dockerfiles are merged and the new `project-clone/Dockerfile` is the `rhel` one :)